bigtable/integration_test.go

```diff
@@ -2142,6 +2152,80 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	}
 }
 
+// TestIntegration_DirectPathFallback tests the CFE fallback when the directpath net is blackholed.
+func TestIntegration_DirectPathFallback(t *testing.T) {
+	ctx := context.Background()
+	testEnv, _, _, table, _, cleanup, err := setupIntegration(ctx, t)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer cleanup()
+
+	if !testEnv.Config().AttemptDirectPath {
+		return
+	}
+
+	if len(blackholeDpv6Cmd) == 0 {
+		t.Fatal("-it.blackhole-dpv6-cmd unset")
```

Comment:
Looks like this will fail the test if the flags aren't passed in. Perhaps we should skip instead?

Reply:
We have that 3 lines above:
```
if !testEnv.Config().AttemptDirectPath {
  return
}
```
If the test want to test DirectPath, then we can fail the test if no command is given.